.github/workflows/semantic.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Semantic Pull Request'
```

Comment:
what does this do?

Reply:
Semantic pull requests and semantic versioning. I'll write a detailed explanation once I'm back. We should probably first discuss release procedures.

Reply:
I've removed the file from this branch and we'll deal with it in the next PR.